src/test/java/com/ays/institution/controller/InstitutionControllerTest.java

```diff
@@ -0,0 +1,96 @@
+package com.ays.institution.controller;
+
+import com.ays.AbstractRestControllerTest;
+import com.ays.common.model.dto.response.AysResponse;
+import com.ays.common.model.dto.response.AysResponseBuilder;
+import com.ays.common.util.exception.model.AysError;
+import com.ays.institution.model.Institution;
+import com.ays.institution.model.dto.response.InstitutionResponse;
+import com.ays.institution.model.entity.InstitutionBuilder;
+import com.ays.institution.model.mapper.InstitutionToInstitutionResponseMapper;
+import com.ays.institution.service.InstitutionService;
+import com.ays.util.AysMockMvcRequestBuilders;
+import com.ays.util.AysMockResultMatchersBuilders;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.web.servlet.request.MockHttpServletRequestBuilder;
+import org.springframework.test.web.servlet.result.MockMvcResultHandlers;
+
+import java.util.Arrays;
+import java.util.List;
+
+
+class InstitutionControllerTest extends AbstractRestControllerTest {
+
+    @MockBean
+    private InstitutionService institutionService;
+
+
+    private final InstitutionToInstitutionResponseMapper institutionToInstitutionResponseMapper = InstitutionToInstitutionResponseMapper.initialize();
+
+
+    private static final String BASE_PATH = "/api/v1/institutions";
+
+    @Test
+    void givenNothing_whenInstitutionStatusActive_thenReturnListInstitutionResponse() throws Exception {
+
+        String mockAccessToken = mockSuperAdminToken.getAccessToken();
+
+        Institution institution1 = new InstitutionBuilder().withValidFields().build();
+        Institution institution2 = new InstitutionBuilder().withValidFields().build();
+
+
+        List<Institution> mockInstitutions = Arrays.asList(
+                institution1,
+                institution2
+        );
+
+        // when
```

Comment:
Aşağıdaki şekilde güncelleyebiliriz;

```suggestion
        // When
```

Reply:
Bu kısmı revise ettim

Reply:
@Rapter1990 bu kısım düzeltilmemiş gibi görünüyor, tekrardan bakabilir misin?